src/Domain/CoordinateMaps/BulgedCube.hpp

```diff
@@ -106,14 +106,14 @@ namespace CoordinateMaps {
  *  \f]
  *
  *  ### The Bulged Face Map
- *  To construct the bulged map we interpolate between this cubical face map
- *  and a spherical face map of radius `radius_of_other_surface`, with the
+ *  To construct the bulged map we interpolate between a cubical face map of
+ *  side length \f$2L\f$ and a spherical face map of radius \f$R\f$, with the
  *  interpolation parameter being \f$s\f$, the `sphericity`.
  *  The surface map for the bulged face lying in the \f$+z\f$ direction is then
  *  given by:
  *
  *  \f[
- *  \vec{\sigma}_{+z}(\xi,\eta) = \left\{(1-s)L + \frac{sR}{\rho}\right\}
+ *  \vec{\sigma}_{+\zeta}(\xi,\eta) = \left\{(1-s)L + \frac{sR}{\rho}\right\}
```

Comment:
Is `\vec{\sigma}` the same as `\vec{x}`?  The previous two maps were labeled by `\vec{x}` and this map appears to be an interpolation between them.  Maybe change equation to `\vec{x}(\xi,\eta) = \vec{\sigma}_{+\zeta}(\xi,\eta) = ...` and say something like "this equation defines \vec{\sigma}_{+\zeta}(\xi,\eta), and we similarly define \vec{\sigma}_{+\eta} and so on by appropriate rotations".